tests/local_clean_urls.php

```diff
@@ -181,16 +181,16 @@ public function test_local_cleanurls_simple() {
         $url = 'http://www.example.com/moodle/course/index.php';
         $murl = new moodle_url($url);
         $clean = $murl->out();
-        $this->assertEquals($clean, 'http://www.example.com/moodle/course/', "Clean: index.php off url");
+        $this->assertEquals('http://www.example.com/moodle/course/', $clean, "Clean: index.php off url");
 
         // Nothing to unclean because these urls will get routed directly by apache not router.php.
 
         $url = 'http://www.example.com/moodle/user/profile.php?id=' . $this->staff->id;
         $murl = new moodle_url($url);
         $clean = $murl->out();
         $this->assertEquals($url, $clean, "Not Cleaned: user profile url with username");
-
         $url = 'http://www.example.com/moodle/user/view.php?id=' . $this->staff->id . '&course=' . $this->course->id;
+
```

Comment:
I fixed the commit comment.

As for those lines, yeah, will keep that in mind. Just that time it was blurring my eyes because two blocks of code corresponding to two different test-cases were separated by line-break on wrong line.